.editorconfig

```diff
@@ -0,0 +1,8 @@
+# See https://EditorConfig.org
```

Comment:
Could you add a comment to clarify that this file is for shfmt

Reply:
done